packages/pub/sentry_dart_plugin/2.2.1.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Sentry Dart Plugin",
+    "canonical": "pub:sentry_dart_plugin",
+    "version": "2.2.1",
+    "package_url": "https://pub.dev/packages/sentry_dart_plugin",
+    "repo_url": "https://github.com/getsentry/sentry-dart-plugin",
+    "main_docs_url": "https://docs.sentry.io/platforms/flutter/upload-debug/"
```

Comment:
```suggestion
    "main_docs_url": "https://pub.dev/documentation/sentry_dart_plugin/"
```

Reply:
since we don't have a specific docs page for it we can use the pub dev docs page, similar to how we use the [rubydocs page for fastlane-plugin](https://github.com/getsentry/sentry-release-registry/blob/f601b502cdd1aa5bd62dc75955d3bd91636da51e/packages/gem/fastlane-plugin-sentry/1.25.1.json#L7)